pull_flutter/lib/views/settings.dart

```diff
@@ -43,6 +39,20 @@ class _PullSettingsPageState extends ConsumerState<SettingsPage> {
                 launchUrl(Uri.parse('https://pulldating.tips'));
               },
             ),
+            ListTile(
+              title: Text('Licenses',
+                  style: GoogleFonts.nunito(
+                      fontWeight: FontWeight.bold,
+                      color: Colors.purpleAccent,
+                      fontSize: 12)),
+              onTap: () {
+                showLicensePage(
+                  context: context,
+                  applicationName: "Pull Dating",
+                  applicationIcon: const ImageIcon(AssetImage("assets/icons/PullLogo.png")),
```

Comment:
Using `ImageIcon` causes the icon to show up in grayscale. Maybe `Image.asset("assets/icons/PullLogo.png", width: 40, height: 40)`?

Reply:
updated the image icon on the license page